src/components/Pages/_EventList.scss

```diff
@@ -0,0 +1,239 @@
+.coa-EventListPage__Entry {
```

Comment:
overall styles question: have we abandoned colors/spacing in sass vars entirely? I know I've been magic numbering it up a bit lately and it doesn't really bother me, just wondering if we're still trying to support a sweeping "make all the blues into purples" design decision in the future or not

Reply:
I want to make an issue for someone to go and clean up all of our sass vars. It would be better to use them.